hooks/_common.sh

```
@@ -206,8 +206,13 @@ function common::get_cpu_num {
 
   local cpu_quota cpu_period cpu_num
 
+  local wslinterop_path = "/proc/sys/fs/binfmt_misc/WSLInterop"
```

Shellcheck (0.10.0)

[error] Remove spaces around = to assign.

(SC2290)